client/src/components/form/UpdateBiodata.tsx

```
@@ -117,19 +131,22 @@ const MultiStepForm: React.FC = () => {
   return (
     <div className="p-4">
       {isSubmitted ? (
-        <div>
-          <h2>Form Submitted Successfully!</h2>
-          <button onClick={handleReset}>Reset Form</button>
-        </div>
+         <div className="flex flex-col items-center p-4 md:p-8 m-4 rounded bg-green-100 gap-2">
+         <Check className='text-white font-bold rounded-full bg-green-900 p-1'/>
+         <h2 className='text-lg md:text-4xl my-2 font-bold text-green-800'>Form Submitted Successfully!</h2>
+         <p className='text-sm font-semibold text-white bg-green-800 rounded p-2'>Visit Your Profile for Checking Update</p>
+         <button onClick={handleReset} className="py-2 px-4 bg-red-700 md:text-lg text-white font-semibold rounded-full shadow-md hover:bg-red-900 focus:outline-none focus:ring focus:ring-offset-2 focus:ring-red-400 focus:ring-opacity-75">Reset</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)